src/module/loader/module-resolver.cpp

```diff
@@ -194,6 +205,12 @@ ModuleInfo ModuleResolver::ModuleResolverImpl::ResolveFromEnv(const filesystem::
     return ModuleInfo{ModuleType::NONE, std::string(), std::string()};
 }
 
+ModuleInfo ModuleResolver::ModuleResolverImpl::ResolveFromContent(const filesystem::Path& name,
+                                                                  const filesystem::Path& path) {
+    auto fullPath = path.IsEmpty() ? path : (path / name).Normalize();
```

Comment:
Good catch, I was thinking about using empty path for ignoring module caching. But later I decided  not to cache module from content, to be consistent with Node. 

So I will just remove the empty check.